power-sequencer/mihawk-cpld.cpp

```diff
@@ -253,24 +332,22 @@ int MIHAWKCPLD::readFromCPLDPSUErrorCode(int bus, int Addr)
         close(fd);
     }
 
-    int statusValue;
-
-    statusValue = i2c_smbus_read_byte_data(fd, StatusReg_2);
+    int statusValue_2;
+    statusValue_2 = i2c_smbus_read_byte_data(fd, StatusReg_2);
     close(fd);
 
-    if (statusValue < 0)
+    if (statusValue_2 < 0)
     {
-        statusValue = 0;
+        statusValue_2 = 0;
     }
 
     // return the i2c-read data
-    return statusValue;
+    return statusValue_2;
 }
 
-// Check for PoweronFault
-bool MIHAWKCPLD::checkPoweronFault()
+// Check PSU_DC_PGOOD state form PSU register via CPLD
+bool MIHAWKCPLD::checkPSUDCpgood(bool polling)
```

Comment:
I think you want this to be an int, since you don't just want a true/false?

Reply:
@spinler Umm...I miss this error which not fixed...
I'll fix it and pull new request again.